openshift/docs/deployment-automation.md

```diff
@@ -0,0 +1,62 @@
+## Github Flow Based Deployments
+
+The deployment pipeline of the microservices in this project are of the Github Flow Type. 
+
+__TL;DR__ 
+- All PRs should be relatively short lived
+- PR code is pushed into a production environment
+- Prior to being merged into master, the source code in master therefor can be defined as always 100% working
+and deployable code. 
+- Each PR will deploy a running instance of the code in the development openshift namespace
+- Each PR's is subsequently promoted to test/prod
+- Not more than 1 PR may be promoted to an environment at the same time
```

Comment:
I don't understand this logic.  Does that mean if one person builds a new pr, it will nuke the old ones?

Reply:
I should clarify for test and prod only. This is similar to any github flow, you wouldn't attempt to promote different integration branches to production at the same time.